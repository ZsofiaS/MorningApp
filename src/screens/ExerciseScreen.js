import React from 'react';
import { View, Text } from 'react-native';

export default function ExerciseScreen() {
  return (
    <View>
      <Text>Great weather today for a run</Text>
    </View>
  )
}
