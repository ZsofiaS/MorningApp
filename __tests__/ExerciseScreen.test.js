import React from 'react';
import renderer from 'react-test-renderer';
import ExerciseScreen from '../src/screens/ExerciseScreen';

test('exercisescreen renders correctly', () => {
  const tree = renderer.create(<ExerciseScreen />).toJSON();
  expect(tree).toMatchSnapshot();
})
