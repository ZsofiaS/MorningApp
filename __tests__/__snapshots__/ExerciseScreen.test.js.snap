// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exercisescreen renders correctly 1`] = `
<View>
  <Text>
    Great weather today for a run
  </Text>
</View>
`;

exports[`homescreen renders correctly 1`] = `
<View>
  <Text>
    Great weather today for a run
  </Text>
</View>
`;
